doc/src/learn/cryptography/sui-multisig.md

```diff
@@ -1,37 +1,34 @@
 ---
 title: Sui Multi-Signature
 ---
+Sui supports `k` out of `n` Multi-Signature (MultiSig) transactions where `k` is the threshold and `n` is the total weights of all participating parties. The maximum number of parties is required to be `<= 10`. 
 
-Sui supports `k` out of `n` multi-weight multi-scheme multi-signature (multisig) transactions where `n <= 10`. 
-A multisig transaction is one that requires more than one private key to authorize it. This topic demonstrates the 
-workflow to create a multisig transaction in Sui, and then submit it using the Sui CLI against a local network. To learn
-how to set up a local network, see [Sui Local Network](../build/sui-local-network.md).
+Pure Ed25519, ECDSA Secp256k1 and ECDSA Secp256r1 are supported as valid participating keys for MultiSig. A ([u8](https://doc.rust-lang.org/std/primitive.u8.html)) weight is set for each participating keys and the threshold can be set as [u16](https://doc.rust-lang.org/std/primitive.u16.html). If the serialized MultiSig contains enough valid signatures of which the sum of weights passes the threshold, the MultiSig is considered valid and the transaction can be executed. 
 
-## Applications of multi-signatures
+This topic discusses the applications of Multi-Signature and the workflow to create a Multi-Signature transaction in Sui.
 
-Multi-signature wallets offer several unique benefits to users, from increased security to escrow transactions and 2FA. 
-Unlike conventional multisig wallets, a Sui multisig account can accept weighted public keys from multiple key schemes.
-Currently, each key can be one of the following schemes: Ed25519, ECDSA secp256k1, and ECDSA secp256r1. 
+# Applications of Multi-Signature
```

Comment:
@kchalkias should we talk about the application first then the cli example ? or we should flip the sections?

Reply:
I was in a dilemma as well, for some reason I prefer the applications first, but let's have a good clear intro that we first present some applications and then documentation.